packages/react/src/runtime/__tests__/style-ssr.test.tsx

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
-import { renderToStaticMarkup } from 'react-dom/server';
+import { renderToStaticMarkup, renderToString } from 'react-dom/server';
 import Style from '../style';
 
 describe('<Style />', () => {
   it('should render style as children on the server', () => {
     const result = renderToStaticMarkup(<Style>{[`.a { display: block; }`]}</Style>);
 
-    expect(result).toInclude('<style>.a { display: block; }</style>');
+    expect(result).toInclude('<style data-cmpld="true">.a { display: block; }</style>');
   });
 
   it('should render style as children on the server with nonce', () => {
     const result = renderToStaticMarkup(<Style nonce="1234">{[`.a { display: block; }`]}</Style>);
 
-    expect(result).toInclude('<style nonce="1234">.a { display: block; }</style>');
+    expect(result).toInclude(
+      '<style data-cmpld="true" nonce="1234">.a { display: block; }</style>'
+    );
+  });
+
+  it('should not output html comments', () => {
```

Comment:
Might be dumb to ask but where this test came from? 🤔

Reply:
when i was running ssr i noticed the html has html comments in it! because we were setting children as an array

i added this with the fix